ocs_ci/ocs/utils.py

```diff
@@ -724,11 +726,13 @@ def collect_ocs_logs(dir_name):
         f"{dir_name}_ocs_logs"
     )
 
-    run_must_gather(os.path.join(log_dir_path, 'ocs_must_gather'),
-                    ocsci_config.REPORTING['ocs_must_gather_image'])
+    if ocs:
+        run_must_gather(os.path.join(log_dir_path, 'ocs_must_gather'),
+                        ocsci_config.REPORTING['ocs_must_gather_image'])
```

Comment:
```suggestion
run_must_gather(
    os.path.join(log_dir_path, 'ocs_must_gather'),
    ocsci_config.REPORTING['ocs_must_gather_image']
)
```

Edited to conform to our style guide. The same should be done to lines 737+738